sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -137,6 +136,47 @@ object CommandUtils extends Logging {
     size
   }
 
+  def calculateLocationsSizes(
+      sparkSession: SparkSession,
+      tid: TableIdentifier,
+      paths: Seq[Option[URI]]): Seq[Long] = {
```

Comment:
do we need to return a Seq[Long]? both the caller sides only need the sum. If we return sum, we can simplify https://github.com/apache/spark/pull/27293/files#diff-cc5307336e7e0368b17c94d8dd7afe41R168